src/linalg/cholesky.rs

```diff
@@ -41,6 +42,7 @@ where
 impl<T: SimdComplexField, D: Dim> Cholesky<T, D>
 where
     DefaultAllocator: Allocator<T, D, D>,
+    T::SimdRealField: Zero,
```

Comment:
This shouldn’t be needed.

Reply:
But I need to create `T::SimdRealField::zero()`. It won't work with `T::zero()`.

Reply:
You can call `T::SimdRealField::zero()` without the explicit `T::SimdRealField: Zero` bound.

Reply:
Oh, right! For some reason I thought that it is necessary.